ups/pex_config.cfg

```diff
@@ -3,7 +3,6 @@
 import lsst.sconsUtils
 
 dependencies = {
-    "required": ["daf_base", "pex_policy"],
```

Comment:
daf_base isn't required anymore because it's solely a runtime-import Python dependency, correct?

Reply:
Yes.